src/jit/importer.cpp

```diff
@@ -12881,7 +12947,9 @@ MATH_MAYBE_CALL_NO_OVF:         ovfl = false;
 
             JITDUMP(" %08X", resolvedToken.token);
 
-            if (!opts.IsReadyToRun())
+            fNeedsRuntimeLookup = info.compCompHnd->computeNeedsRuntimeLookupAndHandleType(&resolvedToken, FALSE);
```

Comment:
I am wondering whether this new method on the JIT-EE interface should be needed at all - the pattern seems to be always:

if (opts.IsReadyToRun())
{
     if (!info.compCompHnd->computeNeedsRuntimeLookupAndHandleType
     {
           info.compCompHnd->getReadyToRunHelper
           ...         
           return; 
     }
     else
     {
           generate the code using the type handle, etc.
     }
}